src/pool/srv_pool.c

```diff
@@ -6975,10 +6973,16 @@ pool_svc_update_map_internal(struct pool_svc *svc, unsigned int opc,
 		goto out_map;
 	}
 
-	/* For SWIM exclude, don't change pool map if the pw_rf is broken or is going to be broken,
-	 * with CRIT log message to ask administrator to bring back the engine.
+	/*
+	 * For SWIM exclude:
+	 * Do not change pool map if the pw_rf is broken or is going to be broken,
+	 * With CRIT log message to ask administrator to bring back the engine.
+	 *
+	 * For DMG exclude:
+	 * Do not change the pool map if the `pw_rf` is broken or is about to break,
+	 * unless the force option is given.
 	 */
-	if (src == MUS_SWIM && opc == MAP_EXCLUDE) {
```

Comment:
the original intention of adding src is not only for this usage.
Will pass that info to rebuild to differenciate err handling. One example is current ds_pool_tgt_revert_rebuild() is incorrect if the rebuild is triggered by CSUM scrubber.

Reply:
@wangshilong, sorry, I didn't know `src` will be used for other purposes, and was only thinking that having both `src` and `skip_rf_check` would create 4 different cases instead of the 2 we currently need. My apologies.

Reply:
No problem, let me add this parameter back for future usage.

Reply:
No problem, looks fine to keep it as it now. I can add it back later when needed, to save retest time.